web/packages/teleport/src/services/instances/index.ts

```diff
@@ -0,0 +1,26 @@
+/**
```

Comment:
is this directory a duplicate of /web/packages/teleport/src/services/instances?, i see index.ts, instances.ts, and types.ts in here

Reply:
It was, this is the correct directory. I deleted the duplicates